app/web/features/search/SearchPage.tsx

```diff
@@ -110,23 +112,17 @@ export default function SearchPage({
       completeProfileFilter,
     ],
     ({ pageParam }) => {
-      // @ts-ignore @TODO David fixing these in a separate PR
-      const lastActiveComparation = parseInt(lastActiveFilter);
-      // @ts-ignore @TODO David fixing these in a separate PR
-      const hostingStatusFilterComparation = parseInt(hostingStatusFilter);
 
+      const numberOfGuestsTemp = numberOfGuestFilter ? parseInt(numberOfGuestFilter) : 0;
```

Comment:
why this:
- I want to display an empty input, if I use the type number (which must be 0 at least) display always shows 0 at the beginning
     - [which isn't an issue but if the user types a number would look like '01' instead of '1'
- making it a string input looks empty, when user deletes the last digit looks empty again (and not a '0') so I think it's better this way even if in the code internally is handled as a string